### ISSUE TITLE
permission error on fuse directories

### ISSUE THREAD

**Issue by mark-hahn** [atom/atom#9555]
I just upgraded to 1.2.0 and now I get a Permission error when trying to open anything, file or project, in my fuse directory.  In particular it is a directory mounted with objectivefs.  This did not happen on 1.1.0 or earlier.

Edit: I just verified it still works on 1.1.0

Atom 1.2.0
Ubuntu 12.04.5 LTS
apm  1.4.0
npm  2.13.3
node 0.10.40
python 2.7.3
git 1.7.9.5

**Comment by kevinsawicki** [comment]
@zcbenz any ideas if this could be related to the Electron upgrade?

**Comment by mark-hahn** [comment]
> any ideas if this could be related to the Electron upgrade?

Or maybe a node upgrade?

**Comment by zcbenz** [comment]
@kevinsawicki It should be related to the Node upgrade in Electron, all the file reading are done on Node's side.

**Comment by mark-hahn** [comment]
> It should be related to the Node upgrade in Electron

I'll do some experiments accessing the fuse directory from the latest
node.  I fear I'm screwed for a while.  I have been telling everyone on the
forum how great objectivefs was.  Oh well ...

**Comment by mark-hahn** [comment]
I have found out that the parents of fuse directories had 600 permissions instead of the usual 755.  This means that any other user wouldn't be able to list the directory I was trying to open with Atom.  But I am always logged in as root and I always start atom with root.  So I think this is the wrong behavior.  Atom shouldn't behave like a different user.  Also it didn't have this problem until after 1.1.0.

In any case I have fixed my problem by setting all dirs to 755.  So I have a workaround.  If anyone wants I'll file another issue about this anomalous behavior but I'm closing this one.
